docs/product-model.md

```diff
@@ -554,10 +536,62 @@ For instance in an FX spot or forward transaction, the respective units
 of the quantity and price will determine who is paying or receiving each
 currency.
 
-## Financial Product
+## Financial Product {#product}
 
 A financial product is an instrument that is used to transfer financial
-risk between two parties. Financial products are represented in the
+risk between two parties. 
```

Comment:
Remove all those unnecessary line breaks ("financial \ risk"). Not sure where they're coming from, but they're peppered across your doc change and make it harder to review.

Reply:
The change is more than a line break; the second sentence has been removed entirely as it implied that financial products can only be represented by the `Product` data type.

Reply:
Ok then, but there are still spurious line breaks all over the shop (including here)